Makefile

```
@@ -1,5 +1,174 @@
-up:
-	docker compose -f docker-compose.yml --env-file=./.env up -d
+.PHONY: help up down restart logs shell bench db-console redis-console test clean up-rebuild down-volumes status ps backup restore dev-setup migrate install-app
```

checkmake (0.2.2)

[warning] Missing required phony target "all"

(minphony)